core/src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala

```diff
@@ -255,8 +175,7 @@ class RemoteServicesMockServer(port:Int) extends RawlsTestUtils {
     mockServer.when(
       request()
         .withMethod("GET")
-        // Apparently the mock server url-decodes paths before comparing
-        .withPath("/ga4gh/v1/tools/#workflow/dockstore-method-path/versions/dockstore-method-version/WDL/descriptor")
+        .withPath("/ga4gh/v1/tools/%23workflow%2Fdockstore-method-path/versions/dockstore-method-version/WDL/descriptor")
```

Comment:
why did this change? This feels like a functional difference.

Reply:
We think there might have been some dependency issue  with regards to taking the womtool library out - I talked to Doug about it and he said not to worry about it